src/convertToObject.js

```diff
@@ -7,6 +7,19 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString
+    .split(';')
+    .reduce((acc, declaration) => {
+      const parts = declaration.split(':');
+      const property = parts[0];
+      const value = parts[1];
```

Comment:
[CODE STYLE] - Avoid variable namings like `parts`, `property`, `value`. These names are too generic and do not describe the data they hold well. Consider naming them in a way that reflects their purpose in the context of CSS styles, such as `styleParts`, `cssProperty`, and `cssValue`.